[2024-11-19T11:04:09.270+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-19T11:04:09.321+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_gbif_data_atropelamento.save_data manual__2024-11-19T11:03:58.178014+00:00 [queued]>
[2024-11-19T11:04:09.356+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_gbif_data_atropelamento.save_data manual__2024-11-19T11:03:58.178014+00:00 [queued]>
[2024-11-19T11:04:09.357+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2024-11-19T11:04:09.399+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): save_data> on 2024-11-19 11:03:58.178014+00:00
[2024-11-19T11:04:09.419+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=398) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-19T11:04:09.427+0000] {standard_task_runner.py:72} INFO - Started process 401 to run task
[2024-11-19T11:04:09.424+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'get_gbif_data_atropelamento', 'save_data', 'manual__2024-11-19T11:03:58.178014+00:00', '--job-id', '726', '--raw', '--subdir', 'DAGS_FOLDER/gbif_dag.py', '--cfg-path', '/tmp/tmpqbbt__8r']
[2024-11-19T11:04:09.428+0000] {standard_task_runner.py:105} INFO - Job 726: Subtask save_data
[2024-11-19T11:04:09.594+0000] {task_command.py:467} INFO - Running <TaskInstance: get_gbif_data_atropelamento.save_data manual__2024-11-19T11:03:58.178014+00:00 [running]> on host 1bd5b698e8d7
[2024-11-19T11:04:09.849+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_gbif_data_atropelamento' AIRFLOW_CTX_TASK_ID='save_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-19T11:03:58.178014+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-19T11:03:58.178014+00:00'
[2024-11-19T11:04:09.851+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-19T11:04:09.852+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-19T11:04:09.853+0000] {logging_mixin.py:190} INFO - Current task name:save_data state:running start_date:2024-11-19 11:04:09.322259+00:00
[2024-11-19T11:04:09.853+0000] {logging_mixin.py:190} INFO - Dag name:get_gbif_data_atropelamento and current dag run status:running
[2024-11-19T11:04:09.853+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-19T11:04:10.115+0000] {minio.py:40} INFO - created gbif_data/atropelamento_data/2024/11/19/atropelamento_occurrences_110402.json object; etag: 2fd37c96c4e868e447b54649dabd7b1f, version-id: None
[2024-11-19T11:04:10.116+0000] {minio.py:42} INFO - Data successfully uploaded to MinIO in bucket environbit under gbif_data/atropelamento_data/2024/11/19/atropelamento_occurrences_110402.json
[2024-11-19T11:04:10.117+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-19T11:04:10.132+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-19T11:04:10.134+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=get_gbif_data_atropelamento, task_id=save_data, run_id=manual__2024-11-19T11:03:58.178014+00:00, execution_date=20241119T110358, start_date=20241119T110409, end_date=20241119T110410
[2024-11-19T11:04:10.153+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-19T11:04:10.154+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-19T11:04:10.154+0000] {logging_mixin.py:190} INFO - Dag name:get_gbif_data_atropelamento queued_at:2024-11-19 11:03:58.195282+00:00
[2024-11-19T11:04:10.155+0000] {logging_mixin.py:190} INFO - Task hostname:1bd5b698e8d7 operator:_PythonDecoratedOperator
[2024-11-19T11:04:10.213+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-19T11:04:10.238+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-19T11:04:10.243+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
